### PR TITLE
[B5] Case Search app second split

### PR DESCRIPTION
**Pull request by jingcheng16**
## Technical Summary

The initial split missed changes made to the `profile_case_search.html` in PR #34400 and PR #34409 
So this PR is a second split based on initial split. 
Please review by commit. It's important to review the [final diff file](https://github.com/dimagi/commcare-hq/blob/b6facd4ce43ff3f425afddde4e6dc05ca04ab543/corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/case_search/profile_case_search.html.diff.txt).

## Feature Flag

(required) Turn on the "[Enable extension sync](https://www.commcarehq.org/hq/flags/edit/extension_sync/?__hstc=240960668.2195ac63c0056c39905e6882865ae8e3.1597171287126.1616447050240.1616526330286.228&__hssc=240960668.8.1616526330286&__hsfp=1126783430)" feature flag

(required) Turn on the "[Enable synchronous mobile searching and case claiming](https://www.commcarehq.org/hq/flags/edit/search_claim/)" feature flag

### Safety story

Tested locally and on staging

### Automated test coverage

We have test for diffs.

### QA Plan

No QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% load compress %}
 {% load i18n %}
 
-{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
+{% requirejs_main_b5 'case_search/js/bootstrap3/profile_case_search' %}
```
Comment (biyeun) [inline corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html:6]:
```suggestion
{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
```

File: corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/case_search/profile_case_search.html.diff.txt [hunk 2]
```diff
@@ -5,22 +5,11 @@
  {% load i18n %}
  
 -{% requirejs_main 'case_search/js/bootstrap3/profile_case_search' %}
-+{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
++{% requirejs_main_b5 'case_search/js/bootstrap3/profile_case_search' %}
```
Comment (biyeun) [inline corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/case_search/profile_case_search.html.diff.txt:8]:
```suggestion
+{% requirejs_main_b5 'case_search/js/bootstrap5/profile_case_search' %}
```

File: corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html:6]: hmm, I should take a look at doing this rename automatically in the tool

Comment (jingcheng16) [reply corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html:6]: Thank you Biyeun! These two changes are flagged in the migration log. 
```
Flagged Code:

{% extends 'hqwebapp/bootstrap3/base_section.html' %}

Guidance:
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
This template extends a bootstrap 3 template.
```
So they should be changed in migrating view stage? (Though it doesn't matter if we change it now)

Comment (orangejenny) [reply corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html:6]: It looks like that flag is slightly different: it's for the `extends`, not the `requirejs_main_b5`. But yeah, it doesn't matter if you make the change now or later.

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html (hunk 1 above)
Comment (jingcheng16) [reply corehq/apps/case_search/templates/case_search/bootstrap5/profile_case_search.html:6]: Thank you @orangejenny . I think Biyeun is correct. I go back to my old initial split PR and realized the reference to b5 js file will happen when the command line tool split js file. I will apply Biyeun's suggestion!

**Review by gherceg — APPROVED** [review]

**Pull request merged**